gcc.symbols.imp

```diff
@@ -145,8 +145,8 @@
   { symbol: [ "ssize_t", private, "<sys/uio.h>", public ] },
   { symbol: [ "ssize_t", private, "<unistd.h>", public ] },
   { symbol: [ "suseconds_t", private, "<sys/types.h>", public ] },
-  { symbol: [ "suseconds_t", private, "<sys/time.h>", public ] },
   { symbol: [ "suseconds_t", private, "<sys/select.h>", public ] },
+  { symbol: [ "suseconds_t", private, "<sys/time.h>", public ] },
```

Comment:
Not so sure about the alpha-ordering. I think the mappings should come in order of preference.

Thinking further about it, though -- I guess it only matters for the first vs the trailing. If no `#include` is present in the source code, IWYU will use the mappings to assign an owning header to a symbol. And in that case it will only use the first available.

I _think_ in all other cases there will be an existing header, and if that appears among the mapped headers, it will be used.

So it seems like order-of-preference only matters for the first vs the rest, and alpha-sorting the rest should be fine.

Reply:
> Not so sure about the alpha-ordering. I think the mappings should come in order of preference.
> 
> Thinking further about it, though -- I guess it only matters for the first vs the trailing. If no `#include` is present in the source code, IWYU will use the mappings to assign an owning header to a symbol. And in that case it will only use the first available.

That's what I did.

> 
> I _think_ in all other cases there will be an existing header, and if that appears among the mapped headers, it will be used.
> 
> So it seems like order-of-preference only matters for the first vs the rest, and alpha-sorting the rest should be fine.